spec/sincerely/services/process_delivery_event_spec.rb

```diff
@@ -36,6 +36,10 @@
 
     context 'when bounce event' do
       let(:event_type) { 'bounce' }
+      let(:event) do
+        double(:event, event_type:, message_id:, recipient:, timestamp:, delivery_system:, options:, # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
Do we want to disable the `RSpec/VerifiedDoubles` cop globally? I think so...

Reply:
> Do we want to disable the `RSpec/VerifiedDoubles` cop globally? I think so...

I think forcing `instance_double` is required, however in these case it is an exception as the event is a polymorphic object